### ISSUE TITLE
Auras

### ISSUE THREAD

**Issue by shexnaex123** [elliotmendiola/WeakAuras#2]
options do show up but i cannot track anything errors just keep popping

**Comment by elliotmendiola** [comment]
Yes this is in very early stages of development... Plus there's a lot of work because all the libraries need to be back ported too... certain custom scripts work and I got it to be very useful but only entirely with custom Scripts

**Comment by elliotmendiola** [comment]
I don't have the bandwidth to work on this right now but if anyone wants to contribute I'm open... this was intended to be an open source back port

**Comment by elliotmendiola** [comment]
A good portion of the default options don't work because events we're dramatically Rewritten by the time the original was released... There's a lot of basic work that will go into making the UI work without using custom Scripts
